### PR TITLE
Lever fast review

### PR DESCRIPTION
**Pull request by hyperwidget**
* This adds in selectors to obfuscate values and the resume header in Lever's new Fast Resume Review flow.

This is _relatively_ imperfect, as we can't update the styling _inside_ the iframe (mostly due CORS) for the resume _and_ there are conditions where resumes either aren't uploaded, or aren't available for preview yet. Ideally we'd have a dynamic selector leveraging the `:has` selector or some dynamicism through JS (this could _potentially_ be a future enhancement, but it'll be hard to add listeners to elements that are rendered by a modern framework, as those elements get created and destroyed frequently - we'll see!).

* Angellist was complaining about some issues due to mixed up constants, so this will fix that too.
* I also made lever's url that it compares against just check for `lever.co` instead of hire/sandbox. This _should_ have two benefits:
  * devs don't have to change the url while working on lever
  * Lever should be able to roll out whichever domains they want as long as they have `lever.co` in them (though, we'll likely still have to update the manifest, or glob the manifest 🤷‍♂️)

lever-fast-resume

image

### CODE REVIEW

File: src/contentScripts/utils/styles.js [hunk 1]
```diff
@@ -21,5 +21,7 @@ export const STYLES = {
   overlay:
     '{ content:"  "; display: block;  height: 120px; position: absolute; top: 120px; left: 0; width: 100%; background-color: rgba(230, 230, 230, 1); z-index: 999; }',
   leverResumeOverlay:
-    '{ content:"  "; display: block;  height: 150px; position: absolute; top:0; left: 0; width: 100%; background-color: rgba(230, 230, 230, 1); z-index: 999; }',
+    '{ content:"  "; display: block;  height: 150px; position: absolute; top:0; left: 0; width: 100%; background-color: rgba(230, 230, 230, 1); z-index: 1; }',
```
Comment (bhavyata9) [inline src/contentScripts/utils/styles.js:24]: The GIF looks great!
- Can we be sure that this is enough? Or it isn't too much?
- Also, I saw that there is a popup on the bottom right in the GIF that says "Skipped Martin Huack". Can we hide the name here as well?

Comment (hyperwidget) [reply src/contentScripts/utils/styles.js:25]: 1. We can't really be sure; we've just got to do our best guess, without some sort of ML approach, we're not really going to be able to have a one size fits all solution for resumes
2. Good catch! I didn't even see that 🙌

Comment (hyperwidget) [reply src/contentScripts/utils/styles.js:25]: (turns out we can't do much about 2 right now - there's not a good way to select that text - so maybe we'll see if we can get a selector for it :D

Comment (bhavyata9) [reply src/contentScripts/utils/styles.js:25]: :( hmm yeah we can tell lever to add a selector for it. Thanks for looking into it!

**Review by bhavyata9 — APPROVED** [review]

**Pull request merged**